lib/puma/minissl.rb

```diff
@@ -5,6 +5,8 @@
 rescue LoadError
 end
 
+# Need for OPENSSL_VERSION and OPENSSL_LIBRARY_VERSION
+require 'openssl'
```

Comment:
@MSP-Greg this look right to you?

Reply:
@nateberkopec

I think all the changes in this file should be removed, as #2305 should have fixed issues when compiled without ssl.